config/open-feature/org.yaml

```diff
@@ -34,6 +35,7 @@ members:
   - agentgonzo
   - ajhelsby
   - alina-v1
+  - AlexJones
```

Comment:
```suggestion
  - AlexsJones
```

you do have a typo in this user name

Reply:
Thank you, thought I had messed other stuff up